### PR TITLE
SAK-25043: Forums/Topics created by group should set student, access, etc. role permissions to 'None'

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-25043

When you create a topic or forum automatically by group, no-one besides the group for that topic/forum should have the "Contributor" permission.

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/jsp/dfReviseTopicSettingsAttach.jsp [hunk 1]
```diff
@@ -44,7 +44,50 @@
...
+	// locate and record SELECT dom nodes with Contributor OPTIONs initially SELECTED
+	function primeAutoCreate() {
+		if (window.originalContributors) {
+			return; // prime only once
+		}
+		window.originalContributors = [];
+
+		// detect which SELECTs will have their value saved.  do detection only once, on page load ...
+		$("span.permissionRow > span > select").each(function() {
+			if ( $(this).children("option:selected[value='Contributor']").length === 1 ) {
```
Comment (bjones86) [inline msgcntr/messageforums-app/src/webapp/jsp/dfReviseTopicSettingsAttach.jsp]: Just a note that 'Contributor' (and 'None' below) does not need to be internationalized, as the `value` attributes are always in English; it's just the display value that's translated:

screenshot_20180303_131730

**Comment by ottenhoff** [comment]
CC: @ansorgej for an opinion?

**Comment by bjones86** [comment]
Waiting on CLA

**Comment by bjones86** [comment]
Thanks, I'll take a look at those properties tomorrow evening.

**Comment by bjones86** [comment]
Yes, the sakai.property is the way! Closing this PR.

**Pull request closed without merging**